### DISCUSSION TITLE
「Markdown 記法チートシート」の「数式の挿入」の警告文について

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#840] [要望・アイデア・不具合]
「Markdown 記法チートシート」の「数式の挿入」の部分に、

> 2023年1月現在、数式表示が崩れるご報告を数件いただいております。表示が崩れる際は、以下の方法を一度お試しください。数式部分で右クリック \> Math Settings \> Math Renderer \> HTML-CSSにチェックが入っている部分を「Common HTML」に変更

という警告文があります。「これにより、いま不満を感じている部分の表示が改善されるのではなかろうか」という淡い期待に誘われて、Qiita の投稿エディタ上で CHTML を SVG に変えてみました（実際には、チートシートの内容とは選択肢が少し異なります）。確かに、幾分変化はしましたが改善とまでは言えません。

　しかし、悪いことに、この状態で（既定のレンダリングに戻せると思って）ブラウザの更新ボタンをクリックしてみたところ、一瞬にして、数式の表示がメチャメチャに崩れてしまいました。この状態で他の投稿記事を閲覧したところ、まともに表示される数式は一つも見つかりません。

　ブラウザのキャッシュをクリアしても直りません。「閲覧履歴」「Cookie と他のサイトデータ」まで削除してやっと直りました。一安心しましたが、かなり慌てました。これは困ります。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
https://github.com/increments/qiita-discussions/discussions/417
上記と同じ内容を「Markdown 記法チートシート」にも記載しておりましたが
こちらを「SVG」でお試しいただいたということですね。

私もtsubolaboさんに記載いただいた内容を試してみたところ、エディタ上の内容については同じような現象になりました。
ただ、私の場合は「閲覧履歴」「Cookie と他のサイトデータ」を削除しなくても直ったのと、他の投稿記事については特に問題はありませんでした。

【戻すためにやった事】
一度エディタを閉じ、適当な記事の数式上で「Math Renderer」の設定を戻し、ブラウザの更新ボタンをクリック

Local storage に「Math Renderer」の設定が保存されていて、それが消えなかったことが原因だったのではないかと考えています。
もし次に「Math Renderer」の設定を変えた場合は、設定をまた元に戻していただくだけで、大丈夫ではないかと思いますが
上記も参考にしていただけると幸いです。

「Markdown 記法チートシート」の記事では「Common HTML」以外に変えることを想定しておりませんでしたが
現在は、「Math Renderer」に 「CHTML」 と 「SVG」 しかないため、記載を削除しておきました🙇
